client/components/suggestion-search/index.jsx

```diff
@@ -148,7 +147,7 @@ class SuggestionSearch extends Component {
 				/>
 				<Suggestions
 					ref={ this.setSuggestionsRef }
-					query={ this.state.query }
+					query={ escapeRegExp( this.state.query ) }
```

Comment:
To prevent the entire site from borking if you enter `(*&^%*^&%$%^&(****^&%^(*^*$*%^%(` or something like that :)